### PR TITLE
Clean up WG-CLI's members

### PR DESCRIPTION
**Pull request by epage**
- @spacekookie: through DM said they were ok to be removed
- @yoshuawuyts: through DM said they were ok to be removed
- @pksunkara: has been acting in the role of a member for a while
- @epage (myself): has been acting as the de facto leader for a while
- deg4uss3r: confirmed over email their continued involvement
- matthiasbeyer: confirmed over email their continued involvement
- Dylan-DPC: conmfirmed over discored their continued involvement

### CODE REVIEW

File: people/pksunkara.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name = "Pavan Kumar Sunkara"
+email = "eopage@gmail.com"
```
Comment (pksunkara) [inline people/pksunkara.toml:2]:
```suggestion
email = "pavan.sss1991@gmail.com"
```

**Review by pksunkara — APPROVED** [review]
Just one comment. Thanks.

File: people/pksunkara.toml (hunk 1 above)
Comment (epage) [reply people/pksunkara.toml]: You can tell what I used as a template :)

**Review by pksunkara — DISMISSED** [review]

**Review by yoshuawuyts — APPROVED** [review]

File: teams/wg-cli.toml [hunk 2]
```diff
@@ -2,18 +2,19 @@ name = "wg-cli"
 kind = "working-group"
 
 [people]
-leads = ["spacekookie"]
```
Comment (rylev) [inline teams/wg-cli.toml:5]: Would love to get @spacekookie's confirmation on this just to make sure everyone is on the same page.

**Comment by rylev** [comment]
@epage looks like this needs a rebase. Since you got @spacekookie to confirm over DM this was ok would you be able to get them to approve this change (for posterity's sake)?

**Comment by epage** [comment]
> @epage looks like this needs a rebase

Rebase done

> Since you got @spacekookie to confirm over DM this was ok would you be able to get them to approve this change (for posterity's sake)?

I PMed them a while ago and they haven't responded.  They mentioned they were stepping back, so its not too surprising if they turned off notifications.

File: people/pksunkara.toml (hunk 1 above)
Comment (pksunkara) [inline people/pksunkara.toml:2]: @epage This got reverted again.

Comment (epage) [reply people/pksunkara.toml]: Thanks for catching that and that is weird that that somehow got reverted when rebasing and resolving conflicts.

**Comment by pksunkara** [comment]
> Since you got @spacekookie to confirm over DM this was ok would you be able to get them to approve this change (for posterity's sake)

Would a screenshot of the DM suffice?

**Comment by rylev** [comment]
I don't feel comfortable sharing screenshots of private correspondence. I think the best thing to do is to merge this. If we hear anything to contrary from @spacekookie we can always figure it out from there.

**Review by rylev — APPROVED** [review]

**Pull request merged**